bin/webpack-dev-server.js

```diff
@@ -26,14 +20,16 @@ const webpack = require('webpack');
 
 const options = require('./options');
 
-const { colors, status, version, bonjour } = require('./utils');
-
 const Server = require('../lib/Server');
 
 const addEntries = require('../lib/utils/addEntries');
+const colors = require('../lib/utils/colors');
+const createConfig = require('../lib/utils/createConfig');
 const createDomain = require('../lib/utils/createDomain');
 const createLogger = require('../lib/utils/createLogger');
-const createConfig = require('../lib/utils/createConfig');
+const getVersions = require('../lib/utils/getVersions');
+const runBonjour = require('../lib/utils/runBonjour');
+const status = require('../lib/utils/status');
```

Comment:
Move this to utils, we should testing their, let's do it in other PR

Reply:
Thank you for separating to utils 🙏